app/models/slot_machine.rb

```diff
@@ -0,0 +1,43 @@
+# A slot machine.
+class SlotMachine < ApplicationRecord
+  has_many :slot_machine_symbols
+
+  # Spins the slot machine, and returns the grid of symbols.
+  #
+  # @return [List[List[Int]]] the grid of symbols.
+  def random_grid
+    rows = 3
```

Comment:
I guess that we will send this value in the future?

Reply:
The slot model does not currently include amount of rows and cols 😿 🎉

Reply:
:neckbeard: